docs/weightonlyquant.md

```diff
@@ -18,6 +18,11 @@ As large language models (LLMs) become more prevalent, there is a growing need f
 |       AWQ      |  &#10004;  | stay tuned |
 |      TEQ      | &#10004; | stay tuned |
 |      GPTQ      | stay tuned | &#10004; |
+
+| Support Device |  PyTorch  |  LLM Runtime  |  RTN  |  AWQ  |  TEQ |  GPTQ  |
+|:--------------:|:----------:|:----------:|:----------:|:----------:|:----:|:----:|
+|     CPU        |  &#10004;  |  stay tuned  |  &#10004;  |  &#10004;  |  &#10004;  |  &#10004;  |

```

Comment:
remove runtime,
gpu didn't support awq now

Reply:
removed， GPU support awq，because inc will output an int4 model with awq, the int4 model is the same with RTN.